src/rvpt/main.cpp

```diff
@@ -29,6 +56,8 @@ int main()
             window.set_close();
         if(window.is_key_down(Window::KeyCode::KEY_R))
             rvpt.reload_shaders();
+        update_camera(window, rvpt);
+        std::cout << rvpt.time.average_frame_time() << '\n';
```

Comment:
Don't spam the console with frame data by default. 
Though, this shouldn't be needed anyways once I get imgui running...